src/app/contact/contact.component.ts

```diff
@@ -18,7 +18,7 @@ export class ContactComponent implements OnInit {
 		this.titleService.setTitle('Contact Loklak');
 	}
 
-	private contactform(event) {
+	public contactform(event) {
```

Comment:
It is not unrelated to the PR, as this updates the version of tslint and codelyzer which now analyzes the errors more closely, and as these methods are used in templates, thus it is required to make this `public`